src/node_metadata.cc

```diff
@@ -34,8 +34,10 @@ std::string GetOpenSSLVersion() {
   // sample openssl version string format
   // for reference: "OpenSSL 1.1.0i 14 Aug 2018"
   char buf[128];
-  const int start = search(OPENSSL_VERSION_TEXT, 0, ' ') + 1;
-  const int end = search(OPENSSL_VERSION_TEXT + start, start, ' ');
+  auto etext = OPENSSL_VERSION_TEXT;
```

Comment:
```suggestion
  const char* etext = OPENSSL_VERSION_TEXT;
```

I would find this helpful especially in the context of this being a change that is about correctness of code.

Reply:
You can t use `+=` operator in this context.

Reply:
@devnexen I don’t see why you can’t do that? (But then again, I don’t see why this would result in a compiler warning either.)

Reply:
Sorry misread your suggestion :-) will make the change